entity-framework/core/modeling/entity-properties.md

```diff
@@ -124,3 +124,16 @@ A property that would be optional by convention can be configured to be required
 [!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/Required.cs?name=Required&highlight=3-5)]
 
 ***
+
+## Column collations
+
+> [!NOTE]
+> This feature is being introduced in EF Core 5.0, which is still in preview.
```

Comment:
You could put this, but then you'll need to create an issue to go back and remove the "is still in preview" part once it no longer is. May be better to just skip the ", which is still in preview." part altogether.

Reply:
I've replaced this with "This feature is introduced in EF Core 5.0." which is what we use elsewhere. I find it grammatically a bit weird, but...

Reply:
Actually it still says "which is still in preview" - which will become out of date at some point? If you're happy to update this later then feel free to ignore this.